lib/screens/authentication/import_key/interactor/mappers/import_key_state_mapper.dart

```diff
@@ -6,22 +6,30 @@ import 'package:seeds/screens/authentication/import_key/import_key_errors.dart';
 import 'package:seeds/screens/authentication/import_key/interactor/viewmodels/import_key_bloc.dart';
 
 class ImportKeyStateMapper extends StateMapper {
-  ImportKeyState mapResultsToState(ImportKeyState currentState, List<Result> results, AuthDataModel authData) {
+  ImportKeyState mapResultsToState({
+    required ImportKeyState currentState,
+    required AuthDataModel authData,
+    required List<Result> results,
+    AuthDataModel? alternateAuthData,
+    List<Result> alternateResults = const [],
+  }) {
+    final List<Result> goodResults = results.isNotEmpty ? results : alternateResults;
```

Comment:
If there are no results for our method then the passport method should have if not it is a total error

Reply:
Yeah